### DISCUSSION TITLE
How can i place the label next to the input?

### DISCUSSION THREAD

**Discussion by silviaAut** [formkit/formkit#692] [General]
Hello,

sorry for that maybe bad question, but I really can't find it in the documentation, but I'm sure I read about it that this is possible.

I want to have the label in one line with the input. Whats the setting for this?

Thanks, Silvia

**Comment by fenilli** [comment]
You should be able to do it using `sections-schema` an changing the schema, or by exporting the input https://formkit.com/guides/export-and-restructure-inputs.
